docs/migrating.md

```diff
@@ -1,5 +1,82 @@
 # Migrating
 
+## Glint 0.9.x to 1.0
+
+{% hint style="warning" %}
```

Comment:
Is this supposed to render when [viewing the file](https://github.com/typed-ember/glint/blob/b0d5bf141a6ac202fb7a287bf2ee6a0d5ab065a0/docs/migrating.md)?

Reply:
Oh wait, I forgot this gets pushed to GitBook. Looks good there.